### PR TITLE
- message bus configuration information update for symfony 4.2+

### PR DESCRIPTION
**Pull request by PanzerLlama**
adds some missing configuration information

### CODE REVIEW

**Comment by ro0NL** [comment]
@PanzerLlama nice, im fine to update the current config block. We dont support SF messenger 4.1 :)

File: docs/cookbook/configuring-a-message-bus.md [hunk 1]
```diff
@@ -36,9 +36,14 @@ framework:
                 middleware:
                     - msgphp.messenger.console_message_receiver
             event_bus:
+                default_middleware: allow_no_handlers
                 middleware:
                     - msgphp.messenger.console_message_receiver
-                    - allow_no_handler
+
+services:
```
Comment (ro0NL) [inline docs/cookbook/configuring-a-message-bus.md]: this should be reverted, as the default behavior is described below. As well as this example snippet.

Comment (PanzerLlama) [reply docs/cookbook/configuring-a-message-bus.md]: yep, sorry

**Review by ro0NL — APPROVED** [review]

**Comment by ro0NL** [comment]
Thanks @PanzerLlama nice catch.

**Pull request merged**